validator/biome.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Same - shouldn't there be one `biome.json` for the whole project, and not one per package?

Reply:
we have specific rules in the webapp too ... will check if I can consolidate them

Reply:
Then leave it for a follow up PR.